action.yml

```diff
@@ -10,40 +10,63 @@ inputs:
     description: 'List of tags that gets passed along to the docker build args'
     required: true
     default: '' # set to an empty string
-  docker_username:
-    description: 'Docker username obtained from secrets'
+  registry_name:
+    description: 'Name of the registry action is loging in to'
     required: true
-  docker_password:
-    description: 'Docker password obtained from secrets'
+  username:
+    description: 'Username of the registry obtained from secrets'
+    required: true
+  password:
+    description: 'Password of the registry obtained from secrets'
     required: true
 
 runs:
   using: "composite"
   steps:
   - uses: actions/checkout@v2
 
+  - name: Get registry url
+    id: get-registry-url
+    run: |
+      url=''
+      if [ "${{ inputs.registry_name }}" = "Docker Hub" ]; then
+        url='docker.io'
+      elif [ "${{ inputs.registry_name }}" = "OSG Harbor" ]; then
+        url='hub.opensciencegrid.org'
+      else
+        echo -e "\e[31mWARNING: Registry not specified\e[0m"
+      fi
+      echo "::set-output name=resitry_url::$url"
+    shell: bash
+
```

Comment:
If we ask for the URL in the variable, we can drop this step. We'll need to update some of the other steps that reference this output as a result